assets/src/components/books/Book.css

```diff
@@ -78,3 +80,21 @@
     width: 100%;
     font-size: 1em;
 }
+
+.book .waitlist-indicator {
+    background-color: var(--book-container-background);
+    color: #999;
+    position: absolute;
+    z-index: 2;
+    right: 0;
+    top: 5px;
+    padding: 2px 6px;
+    font-size: 0.8em;
+    border-radius: 0 0 0 4px;
+}
+
+.book .waitlist-indicator .material-icons {
+    font-size: 15px;
```

Comment:
Can we make all the font-sizes use the same metric? I think we're using px.

Reply:
All changed to em in 21ece60